app/src/neptune_eye/config.py

```
@@ -0,0 +1,223 @@
+    """
+    Validate configuration values for logical consistency.
+    
+    Args:
+        config: Configuration object to validate.
+        
+    Raises:
+        ValueError: If configuration values are invalid or inconsistent.
+    """
+    # Validate confidence threshold
+    if not 0.0 <= config.model.confidence <= 1.0:
+        raise ValueError(f"Model confidence must be between 0.0 and 1.0, got {config.model.confidence}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)